go/host/in_mem_obscuro_client.go

```diff
@@ -35,12 +38,11 @@ func (c *inMemObscuroClient) Call(result interface{}, method string, args ...int
 		if len(args) != 1 {
 			return fmt.Errorf("expected 1 arg to %s, got %d", rpcclientlib.RPCSendRawTransaction, len(args))
 		}
-		tx, ok := args[0].(common.EncryptedParamsSendRawTx)
-		if !ok {
-			return fmt.Errorf("arg to %s was not of expected type EncryptedParamsSendRawTx", rpcclientlib.RPCSendRawTransaction)
+		b, err := json.Marshal(args)
```

Comment:
@BedrockSquirrel Can we use `bytes` instead of `b` as the variable name, here and below? We also use `b` for blocks, it's too opaque.